lib/ui/home/HomeDailyIlliniWidget.dart

```diff
@@ -392,8 +400,11 @@ class _MinorStory extends StatelessWidget {
               Divider(color: Styles().colors!.blackTransparent06),
               Padding(
                   padding: EdgeInsets.only(top: 4, bottom: 8, left: 20, right: 20),
-                  child: Text(StringUtils.ensureNotEmpty(illiniItem?.title), textAlign: TextAlign.left,
-                      style: Styles().textStyles?.getTextStyle('widget.title.medium.extra_fat'))),
+                  child: InkWell(
+                      child: Text(StringUtils.ensureNotEmpty(illiniItem?.title), textAlign: TextAlign.left,
+                          style: Styles().textStyles?.getTextStyle('widget.title.medium.extra_fat')),
+                      onTap: () => launchUrlString(StringUtils.ensureNotEmpty(illiniItem?.link))
```

Comment:
The same:

This opens black screen in Android devices. Please use the following appraoch.

```
  void _onTap() {
    String? url = illiniItem!.link;
    if (StringUtils.isNotEmpty(url)) {
      Uri? uri = Uri.tryParse(url!);
      if (uri != null) {
        LaunchMode launchMode = Platform.isAndroid ? LaunchMode.externalApplication : LaunchMode.platformDefault;
        launchUrl(uri, mode: launchMode);
      }
    }
  }
```

Reply:
fixed